### PR TITLE
Added custom placeholder option

### PR DESCRIPTION
**Pull request by wsheung**
#143 

add name placeholder customization

updated Readme.md

### CODE REVIEW

**Comment by wsheung** [comment]
@shortdiv  Just want to pop in and see if there is an update to this 👍, I am happy to refactor the code if needed.

**Comment by shortdiv** [comment]
Nicely done @wsheung! 

Can you update [the README](https://github.com/netlify/netlify-identity-widget#netlifyidentityinitopts) so users know they can pass this in via options in Netlify Identity. Something like this: 

```
{
  container: '#some-query-selector'; // container to attach to
  APIUrl: 'https://www.example.com/.netlify/functions/identity'; // Absolute url to endpoint.
  namePlaceholder: "nombre"
}
```

File: README.md [hunk 1]
```diff
@@ -128,6 +128,7 @@ module API. Options include:
 {
   container: '#some-query-selector'; // container to attach to
   APIUrl: 'https://www.example.com/.netlify/functions/identity'; // Absolute url to endpoint.  ONLY USE IN SPECIAL CASES!
+  namePlaceholder: 'some-placeholder-for-Name'; // custom placeholder for name input form
 }
```
Comment (wsheung) [inline README.md]: @shortdiv I believe the readme is already updated with the new option but I can definitely change the example to use "nombre" instead if you want

Comment (shortdiv) [reply README.md]: Ooof, I somehow missed that. 🤦🏾 Merging.

**Pull request merged**